account-kit/wallet-client/src/client/actions/signPreparedCalls.ts

```diff
@@ -53,23 +54,40 @@ export async function signPreparedCalls(
     call: PreparedCall_UserOpV060 | PreparedCall_UserOpV070,
   ) => {
     const { signatureRequest, ...rest } = call;
+
+    if (!signatureRequest) {
+      throw new InvalidRequestError({
```

Comment:
❓ nit: do you think it's appropriate to throw an rpc error here when the error is from the local action?

Reply:
Nope, definitely should change. Do we have custom errors or is the generic `Error` better for this case?

Reply:
i would probably use `BaseError` (from our pkg, not viem), unless we have a specific "invalid request" variant already.